README.md

```diff
@@ -78,7 +78,24 @@ terminus secret:delete <site> foo
 
 1. Set the secret value to the token via terminus: `terminus secret:set <site> github-oauth.github.com <github_token> --type=composer --scope=user,ic`
 
-1. Commit a change to generate a new build 
+1. Add your private repository to the `repositories` section of `composer.json`:
+
+    ```json
+    {
+        "type": "vcs",
+        "url": "https://github.com/your-organization/your-repository-name"
+    }
+    ```
+
+    Your repository should contain a `composer.json` that declares a package name in its `name` field and a `type` of `drupal-module` or `wordpress-plugin`, as appropriate for your CMS. For these instructions, we will assume your package name is `your-organization/your-package-name`.
```

Comment:
type is not 100% required as you may be requiring a "regular" composer package/library.

Other than that, it LGTM